CharlieBackend.Api/Controllers/SchedulesController.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CharlieBackend.Business.Services.Interfaces;
+using CharlieBackend.Core.DTO.Schedule;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using CharlieBackend.Core;
+using CharlieBackend.Core.Models.ResultModel;
+
+namespace CharlieBackend.Api.Controllers
+{
+    [Route("api/schedules")]
+    [ApiController]
+    public class SchedulesController : ControllerBase
+    {
+        public readonly IScheduleService _scheduleService;
+        public SchedulesController(IScheduleService scheduleService)
+        {
+            _scheduleService = scheduleService;
+        }
+
+        [Authorize(Roles = "3,4")]
```

Comment:
```suggestion
        [Authorize(Roles = "Secretary, Admin")]
```
Roles should be like this now